.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: 'airbnb-base',
+  rules: {
+    'no-console': 'off',
```

Comment:
I think explicitly disable it per session/line is better, because people will use `console.log` to debug and forget to remove